README.md

```diff
@@ -0,0 +1,290 @@
+# Composable rules
+
+Imagine you need to write rules to modify some data based on a growing set of
+a growing set of business logic.
+You could write it in vanilla JavaScript but chances are you quickly end up with something like this:
+
+```javascript
+if(businessCondition1) {
+  if(businessCondition2) {
+    if(businessCondition3) {
+      if(businessCondition3) {
+        if(businessCondition4) {
+           // do something
+           // KA-ME-HA-ME-HA of death
+        }
+      }
+    }
+  }
+}
+```
+
+![kamehameha](https://media1.giphy.com/media/oTjoawKEq3wYD5fKEh/giphy.gif)
+
+This is a small zero-dependency library created to write rule logic
+to manipulate data that's much more maintainable, readable, composable and reusable
+than writing plain `if/else` logic.
+
+## Basics
+
+The basic structure of a rule the following:
```

Comment:
The basic structure of a rule is the following:

Reply:
Thanks. Fixed